app/src/androidTest/java/hu/vmiklos/plees_tracker/MainActivityInstrumentedTest.kt

```diff
@@ -28,6 +28,7 @@ import java.io.File
 import java.util.Calendar
 import kotlinx.coroutines.runBlocking
 import org.junit.Assert.assertEquals
+import org.junit.Before
```

Comment:
Now that you touch this androidTest, covering the import-from-file scenario, do you think it would be possible to cover the import-from-calendar scenario as well, with reasonable amount of effort?

Reply:
To be honest, not my area of expertise. Would appreciate some help there